packages/edit-site/src/components/page-pages/index.js

```diff
@@ -31,6 +32,12 @@ export default function PagePages() {
 			field: 'date',
 			direction: 'desc',
 		},
+		fields: {
+			// All fields are visible by default, so it's
+			// better to keep track of the hidden ones.
+			hidden: new Set( [ 'date' ] ),
```

Comment:
Should we use a regular array to keep this view config object serializable (if parsed from JSON, there's no need to do anything, it just works)

Reply:
Good thought for persisting the view configs later. I'll update.